modules/vaos/app/services/vaos/systems_service.rb

```diff
@@ -109,11 +109,24 @@ def get_facility_visits(system_id, facility_id, type_of_care_id, schedule_type)
           'institution-code' => facility_id,
           'clinical-service' => type_of_care_id
         }
-        response = perform(:get, url, url_params, headers(@user))
+        response = perform(:get, url, url_params)
```

Comment:
is headers not needed for this request? howcome?

Reply:
good catch, updated